locales/en-US/notes.properties

```diff
@@ -75,7 +75,7 @@ savedComplete2=Saved at {time}
 
 # LOCALIZATION NOTE (syncComplete2): {time} is the time of syncing. Such as 5:33pm or 17:33. If this
 # structure doesn't work for your locale, you can translate this as "Synced ({time})".
-syncComplete2=Synced at {time}
+syncComplete2=Synced
```

Comment:
You can't change an existing string like this, you need a new string ID (and to update the reference in the note)
https://developer.mozilla.org/en-US/docs/Mozilla/Localization/Localization_content_best_practices#Changing_existing_strings

Reply:
@flodolo adding something like *syncComplete3=Synced* would do?

Reply:
Yes. Note that you'd need to remove the existing string if it becomes unused.